docs/workflows/transitions/preprocess/preprocess/utils.py

```diff
@@ -155,6 +157,8 @@ def merge_lines_from_different_pages(predictions, field_config):
         if p['label'] in line_labels:
             line_predictions[p['label']].extend(p['value'])
 
+    line_fields = get_line_labels(field_config)
```

Comment:
For an untrained eye it is hard to see the difference between the variable line_labels assigned in the top of the function and line_fields which is defined here. I suppose line_labels is the "top-level" label name, while line_fields is the columns? 

Maybe give this variable column_names? 

It also seems like we should get column names for the separate line-items? In case a customer has two line fields with the same column.